tester_app/client/app/bundles/HelloWorld/reducers/index.jsx

```diff
@@ -0,0 +1,10 @@
+import helloWorldReducer from './helloWorldReducer';
```

Comment:
// This file is our manifest of all reducers for the app.
// See /client/app/bundles/HelloWorld/store/helloWorldStore.jsx